crates/bevy_color/src/color_gradient.rs

```diff
@@ -0,0 +1,184 @@
+use crate::{ColorRange, Mix};
+use bevy_math::curve::{Curve, Interval};
+
+/// Represents a gradient of a minimum of 1 up to arbitrary many colors. Supported colors have to
+/// implement [`Mix`] and have to be from the same color space.
+///
+/// By default the color values are linearly interpolated.
```

Comment:
`Mix` interpolation is not necessarily linear, so this is kind of misleading (although maybe this language is just borrowed from `ColorRange`, which should probably also be changed...)